File: server/llm/prompt_utils.py
import json


def format_offer_json(offer: dict) -> str:
    return json.dumps(
        {
            "x_hydro": round(float(offer["hydro"]), 2),
            "x_agri": round(float(offer["agri"]), 2),
            "x_infra": round(float(offer["infra"]), 2),
        },
        indent=2,
    )

File: server/llm/promptUtils.js
function formatOfferJson(offer) {
  return JSON.stringify(
    {
      x_hydro: Number(offer.hydro.toFixed(2)),
      x_agri: Number(offer.agri.toFixed(2)),
      x_infra: Number(offer.infra.toFixed(2)),
    },
    null,
    2
  );
}

module.exports = { formatOfferJson };
